gravitee-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/processor/ResponseProcessorChainFactory.java

```diff
@@ -34,7 +34,7 @@
     @Autowired
     private ReporterService reporterService;
 
-    public Processor<ExecutionContext> create() {
+    public Processor<ExecutionContext> create(boolean logRequestBody) {
```

Comment:
What's the purpose of this boolean here?

Reply:
whoops, shame on me. Merge conflict resolution. Fixed.